synapse/http/matrixfederationclient.py

```diff
@@ -339,9 +341,26 @@ def _send_request(
         else:
             query_bytes = b""
 
-        headers_dict = {b"User-Agent": [self.version_string_bytes]}
+        # Retreive current span
+        scope = opentracing.tracer.start_active_span(
```

Comment:
Do we want to have a span within the retry loop too?

Reply:
Could help if the retries are failing regularly but not consistently?